File: ambeth/Ambeth.IoC.Test/testutil/TestFrameworkModule.cs
using System;

namespace De.Osthus.Ambeth.Testutil
{
    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, Inherited = true, AllowMultiple = true)]
    public class TestFrameworkModule : Attribute
    {
        public Type[] Value { get; private set; }

        public TestFrameworkModule(params Type[] value)
        {
            this.Value = value;
        }
    }
}

File: jambeth/jambeth-ioc-test/src/test/java/de/osthus/ambeth/testutil/TestFrameworkModule.java
package de.osthus.ambeth.testutil;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

import de.osthus.ambeth.ioc.IInitializingModule;

@Retention(RetentionPolicy.RUNTIME)
@Target({ ElementType.METHOD, ElementType.TYPE })
public @interface TestFrameworkModule
{
	Class<? extends IInitializingModule>[] value();
}
